geolocation-API/getCurrentPosition_permission_deny.https.html

```diff
@@ -15,20 +15,21 @@
     onSuccess, onError, hasMethodReturned = false;
 
 t.step(function() {
-  onSuccess = t.step_func(function(pos) {
+  onSuccess = t.step_func_done(function(pos) {
```

Comment:
Or I can just do:
`t.unreached_func('A success callback was invoked unexpectedly');`
and ignore the anyway unexpected position.  Either way, let me know.